app/delivery/views/meteors/meteors_view.njk

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title> Meteors </title>
+</head>
+<body>
```

Comment:
you can create layout.njk template and reuse it in several places

Reply:
Updated